batcher/aligned/src/main.rs

```diff
@@ -58,6 +60,11 @@ pub enum AlignedCommands {
     GetUserBalance(GetUserBalanceArgs),
     #[clap(about = "Get user nonce from the batcher", name = "get-user-nonce")]
     GetUserNonce(GetUserNonceArgs),
+    #[clap(
+        about = "Get the first user nonce from ethereum",
+        name = "get-user-first-nonce"
+    )]
+    GetUserFirstNonce(GetUserFirstNonceArgs),
```

Comment:
I'd rename this to something like: `GetUserFirstNonceInBatcherQueue`. The way it is now, I feel it isn't very descriptive.

Reply:
Also, write a more detailed `about`.